packages/components/src/components/ColorPicker/utils.ts

```diff
@@ -0,0 +1,73 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { type ColorFormat, type RgbaColor } from './types';
+
+export const DEFAULT_COLOR = { red: 255, green: 255, blue: 255, alpha: 1, name: '' };
+export const DEFAULT_FORMAT = 'HEX';
+
+export const colorToCss = (color?: RgbaColor) => {
+    if (!color) {
+        return undefined;
+    }
+    return `rgba(${color.red}, ${color.green}, ${color.blue}, ${color.alpha || 1})`;
+};
+
+export const isValidHexColor = (color: string): boolean => {
+    const hexRegex = /^([\dA-Fa-f]{3}){1,2}$/;
+    return color.length > 2 && hexRegex.test(color);
+};
+
+export const hexColorToRgba = (hex: string): RgbaColor => {
```

Comment:
idk about you guys but I'm a big fan of annotating functions like mad

```suggestion
/**
 * Converts a hexadecimal color code to an RGBA color object. This function accepts
 * both three-digit and six-digit hex codes without the '#' symbol.
 * @param {string} hex - The hex code, either 3 or 6 characters long. Characters should be from the set [0-9A-Fa-f].
 * @returns {RgbaColor} 
 * @example
 * hexColorToRgba('FFF'); // { red: 255, green: 255, blue: 255, alpha: 1 }
 * @example
 * hexColorToRgba('FF5733'); // { red: 255, green: 87, blue: 51, alpha: 1 }
 * @example
 * hexColorToRgba('XYZ'); // { red: 0, green: 0, blue: 0, alpha: 0 }
 */
export const hexColorToRgba = (hex: string): RgbaColor => {
```

Reply:
added jsdoc